amo/blockchain/hibernate.go

```diff
@@ -45,24 +47,36 @@ func (m MissRuns) UpdateMissRuns(height int64, vals []crypto.Address) (doValUpda
 	defer batch.Close()
 
 	unfinishedRuns := [][]byte{}
+	finishedRuns := [][]byte{}
+
 	itr, err := runDB.Iterator(nil, nil)
 	if err != nil {
 		return
 	}
 	defer itr.Close()
 	// guard for rewind
```

Comment:
We need to change this comment. This code block is more than just `rewind guard` now.